src/components/React/Layout.js

```diff
@@ -1,207 +1,66 @@
-                University and affiliated RI institutions.<br/>
-              </p>
-              <p>
-                This work was supported by the NSF EPSCoR grant 1655221: "RII Track-1:
-                Rhode Island Consortium for Coastal Ecology Assessment, Innovation, and
-                Modeling (C-AIM)"<br/>
-              </p>
-            </div>
-          </Flexbox>
-        </Element>
+              <h2 className="mt-4 text-center">
+                Learn How To Use the Volume Renderer
+              </h2>
+              <Button variant="primary" onClick={() => setShowHowto(true)}  className="mb-4">Launch Instructions</Button>
 
-        <Element name="peopleTarget" id="peopleTarget" className="element light-page-new">
-          <div id="peopleTarget" className="light-page-new">
-            <Flexbox flexDirection="column" alignContent="center">
-
-              <div className="light-page-title" key="title">
-                <p>People</p>
-              </div>
-
-              <Flexbox justifyContent="space-around">
-                <Flexbox flexDirection="column">
-                  <img src={Camilo} alt="Camilo" height="200px" />
-                  <a className="peopleLink" href="mailto:camilo_diaz@brown.edu?Subject=Interest in AFrame Viewer" target="_top">Camilo Diaz</a>
-                  <a className="peopleLink" href="https://github.com/kmilo9999" target="_blank" rel="noopener noreferrer"><Github width={30} /></a>
-                </Flexbox>
-
-                <Flexbox flexDirection="column">
-                  <img src={Ben} alt="Ben" height="200px" />
-                  <a className="peopleLink" href="mailto:benjamin_knorlein@brown.edu?Subject=Interest in AFrame Viewer" target="_top">Benjamin Knörlein</a>
-                  <a className="peopleLink" href="https://github.com/BenKnorlein" target="_blank" rel="noopener noreferrer"><Github width={30} /></a>
-                </Flexbox>
-              
-                <Flexbox flexDirection="column">
-                  <img src={Kalvin} alt="Kevin" height="200px" />
-                  <a className="peopleLink" href="mailto:kalvin_lam@brown.edu?Subject=Interest in AFrame Viewer" target="_top">Kalvin Lam</a>
-                  <a className="peopleLink" href="https://github.com/theklam" target="_blank" rel="noopener noreferrer"><Github width={30} /></a>
-                </Flexbox>
-              </Flexbox>
-            </Flexbox>
-          </div>
-        </Element>
-
-        <Element name="acknowledgements" className="element">
-          <div id="acknowledgements" className="light-page">
-             <div className="light-page-title" key="title">
-              <p>Acknowledgements</p>
-             </div>
+              <Howto
+                show={showHowto}
+                close={() => setShowHowto(false)}
+              />
+            </div>
+          </Col>
+        </Row>
+      </Container>
```

Comment:
I think if you're gonna use the bootstrap components they should be used everywhere or nowhere. What do you think?

Reply:
My general preference is nowhere or only when they add a lot of functionality, but as they're already in place they should be used where there is an easy drop in.

Reply:
Eternal battle, but I'd vote for consistency... I think if we're using components in most places, we should try to just stick to components. I don't love the readability of mixing the styles / components, and why depend on a component library and not take advantage of the shorter syntax?

Reply:
I guess in this specific case, could we be using [`<Container>`](https://react-bootstrap.github.io/layout/grid/)?

Reply:
for which thing?

Reply:
typically, you don't nest containers

Reply:
Sorry I think this was all in reference to mixing up lines like
`<div className="d-flex justify-content-center m-3">`
and direct Bootstrap componets ... so I was referring to changing that `d-flex` to be a `<Container>` ...

Reply:
Do they make a difference for mobile?